platform/android/java/gradle.properties

```diff
@@ -14,7 +14,7 @@ android.useAndroidX=true
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx4536m
+org.gradle.jvmargs=-Xmx8536m
```

Comment:
build fails with symbols on without this

Reply:
Hmm.. local build of the android will just fails without that bump, but it can be changed locally

Reply:
Instructions can be added for those interested in building the editor so they can update their memory settings accordingly. 
Given this config also affects the other modules, bumping it may affect other developers that are not interested in the building the editor.

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2.

fyi, I've been able to build fine with the original values.